### ISSUE TITLE
About the hyper-parameters of sphereface2 used for ReDimNet

### ISSUE THREAD

**Issue by Hunterhuan** [IDRnD/ReDimNet#10]
Hello, ReDimNet is a very valuable work, and thank you very much for your open source sharing of the code.

I saw in the paper that using sphereface 2 as the loss function would yield better results. May I ask what specific hyper-parameters you used when adopting sphereface 2? (including margin, t, scale, and lambuda for both pretraining and LMF stages.)

Thanks again

**Comment by Hunterhuan** [comment]
Sorry, I noticed that you have already shared the relevant parameters in other issues. Thanks again.

https://github.com/IDRnD/ReDimNet/issues/9
